quarkus-integration/quarkus/integration-test/src/main/java/ai/timefold/solver/quarkus/it/TimefoldTestResource.java

```diff
@@ -39,4 +39,27 @@ public String solveWithSolverFactory() {
             throw new IllegalStateException("Solving failed.", e);
         }
     }
+
+    @POST
+    @Path("/solver-factory/builder")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String solveWithSolverBuilderFactory() {
```

Comment:
The intention of this method was to test the override. Quarkus has some very special logic that deals with terminations (you'll find out very soon), and we want to ensure that if we provide the termination override, things work as expected.